views/includes/metaHead.njk

```diff
@@ -24,5 +24,10 @@
     if (sessionStorage.fontsLoaded) {
         docEl.className += ' ' + 'fonts-loaded';
     }
+
+    if (window.location.protocol === 'file:') {
+        alert("It looks like you've saved a copy of this webpage to work with on your own computer. This will not work correctly and your data will be lost if you attempt to complete this form. Please return to our website at https://www.tnlcommunityfund.org.uk to continue applying.");
```

Comment:
Do we need this in addition to the message? Saving a web page offline isn't unreasonable, feels a bit much to alert on every page.

Reply:
Oops, this was an initial version of this change which I thought I'd reverted! Just removed it.